### ISSUE TITLE
PropTypes

### ISSUE THREAD

**Issue by conradoramalho** [romac/react-if#29]
Can you include array in Then or Else proptypes?

Then.propTypes = Else.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.func,
    PropTypes.string,
    PropTypes.number,
    PropTypes.arrayOf(PropTypes.element),
    PropTypes.object
  ])
};

My problem:

 ```

<If condition={true}>
  <Then>
    <Loading   />
  </Then>
  <Else>
    <div></div>
    <OtherComponent/>
  </Else>
</If>

```

**Comment by conradoramalho** [comment]
I opened this pull request with array proptype
#30

**Comment by meszaros-lajos-gyorgy** [comment]
Yep, I saw it, though it addresses the same issue as what is described in the other ticket.

**Comment by meszaros-lajos-gyorgy** [comment]
I'm working on making the build process actually run, but the code is heavily outdated and some dependencies need major updates. I'll push a pull request soon and you could merge that to yours, so that the build will pass for your branch.

**Comment by conradoramalho** [comment]
Ok, thanks. If you need I can help you 👍

**Comment by meszaros-lajos-gyorgy** [comment]
I've opened a separate issue to address the build problems: #33 
Also, I've linked my pull request there.

**Comment by romac** [comment]
Fixed in #31.
